frontend/src/routes/LandingPage/index.js

```diff
@@ -16,124 +16,122 @@ import AdmissionCountDown from "../../components/AdmissionCountDown";
 
 import LoadingBall from "src/components/LoadingBall";
 
-class LandingPage extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      results: undefined,
-      admission: null,
-      error: null,
-      isLoading: true,
-      hasSubmitted: false,
-    };
-  }
+const LandingPage = () => {
+  const [state, setState] = useState({
```

Comment:
Again, imo. we should split the state into multiple `useState`s, this would avoid the 
`(prevState) => ({ ...prevState, ... })` that is used a lot here.